crates/database/src/orders.rs

```diff
@@ -292,6 +352,9 @@ pub struct FullOrder {
     pub buy_token_balance: BuyTokenDestination,
     pub presignature_pending: bool,
     pub is_liquidity_order: bool,
+    pub pre_interactions_tos: Vec<Address>,
+    pub pre_interactions_values: Vec<BigDecimal>,
+    pub pre_interactions_data: Vec<Vec<u8>>,
```

Comment:
Is there any way in `sqlx` to group this:
```suggestion
    pub pre_interactions: Vec<PreInteraction>,
```

It would be cool if there was!

Reply:
I searched over 1 day for it... Could not find it.

Reply:
This was my worry - that it isn't supported. Maybe @vkgnosis knows a way to do this as he has quite a bit of `sqlx` experience.

Reply:
> It's not really incurring deep technical depth.

Like you said, the "ugliness" is super localized to a specific module and doesn't spread through the code, so its very little debt that, if we are careful, won't grow over time.

Reply:
I don't know a way from the top of my head.